src/os/ObjectStore.cc

```diff
@@ -36,8 +37,22 @@ std::unique_ptr<ObjectStore> ObjectStore::create(
     return std::make_unique<MemStore>(cct, data);
   }
 #if defined(WITH_BLUESTORE)
-  if (type == "bluestore" || type == "random") {
+  if (type == "bluestore") {
     return std::make_unique<BlueStore>(cct, data);
+#ifndef WITH_SEASTAR
```

Comment:
Is there a point in shipping stable bluestore within experimental OSD? Maybe we should switch to `bluestore-exp` for crimson in medium term.

Reply:
I think crimson development would appreciate stability of `bluestore`.
On the other hand it might come a time that we would like to get some extra performance for `bluestore-exp`.
The only option that does not make sense for `crimson` is `random.

Reply:
> I think crimson development would appreciate stability of `bluestore`.

+1